AEPCore/Tests/EventHubTests/EventHubTests.swift

```diff
@@ -1094,4 +1094,45 @@ class EventHubTests: XCTestCase {
         // verify
         validateSharedState(EventHubTests.MOCK_EXTENSION_NAME, event, "one", .xdm)
     }
+
+    // MARK: isExtensionRegistered() Tests
+
+    /// Ensures that false is returned when an non-registered extension is passed in
+    func testIsExtensionRegisteredInvalid() {
+        // setup
+        eventHub.start()
+
+        // test & verify
+        XCTAssertFalse(eventHub.isExtensionRegistered(extensionName: "com.adobe.test.extension"))
+    }
+
+    /// Ensures that true is returned when a registered extension name is passed in
+    func testIsExtensionRegisteredValid() {
+        // setup
+        eventHub.start()
+
+        // test & verify
+        // MockExtension is registred in setup()
+        XCTAssertTrue(eventHub.isExtensionRegistered(extensionName: MockExtension(runtime: TestableExtensionRuntime()).name))
+    }
+
+    /// Tests that isExtensionsRegistered returns the correct value for an extension that is registered, then unregistered
+    func testIsExtensionRegisteredRoundTrip() {
```

Comment:
nit: "RoundTrip" isn't very self explanatory. Maybe `testIsExtensionRegisteredReturnsTrueThenFalseAfterUnregistration` ? I know, verbose, but it's more descriptive.